app/js/App.js

```diff
@@ -20,11 +20,11 @@ class App {
    * Start application by creating a page
    */
 
-  constructor() {
+  constructor(theme = 'default') {
```

Comment:
Yeah, in retrospect I am not sure entirely what I was thinking there. In C#/Java having a separate factory method would make much more sense, but the arguments that apply there don't really apply to JavaScript. I'll push a commit changing it to the more sensible way tomorrow morning.